src/app/pages/dashboard/components/dashboard-form/dashboard-form.component.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ChangeDetectorRef,
+} from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { FormBuilder, FormControl } from '@ngneat/reactive-forms';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import { TranslateService } from '@ngx-translate/core';
+import { Subject } from 'rxjs';
+import { CoreEvent } from 'app/interfaces/events';
+import { FormErrorHandlerService } from 'app/pages/common/ix-forms/services/form-error-handler.service';
+import { DashConfigItem } from 'app/pages/dashboard/components/widget-controller/widget-controller.component';
+import { WebSocketService } from 'app/services';
+import { IxSlideInService } from 'app/services/ix-slide-in.service';
+
+@UntilDestroy()
+@Component({
+  selector: 'app-dashboard-form',
+  templateUrl: './dashboard-form.component.html',
+  styleUrls: ['./dashboard-form.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DashboardFormComponent {
+  title = 'Dashboard Configuration';
+
+  formGroup: FormGroup = this.formBuilder.group({
+    name: ['default', []],
+  });
+
+  dashState: DashConfigItem[] = [];
+  systemWidgets: DashConfigItem[] = [];
+  storageWidgets: DashConfigItem[] = [];
+  networkWidgets: DashConfigItem[] = [];
+
+  isFormLoading = false;
+  formEvents$: Subject<CoreEvent>;
+
+  /* tooltips: BootenvTooltip = {
+    name: helptextSystemBootenv.create_name_tooltip,
+  }; */
```

Comment:
Please confirm to use form indicator, and remove or set the value of `isFormLoading` (also in template)
Remove commented code.